File: app/src/main/java/com/ghstudios/android/features/quests/QuestListPagerActivity.java
package com.ghstudios.android.features.quests;

import com.ghstudios.android.loader.QuestListCursorLoader;
import com.ghstudios.android.mhgendatabase.R;
import com.ghstudios.android.BasePagerActivity;
import com.ghstudios.android.MenuSection;

public class QuestListPagerActivity extends BasePagerActivity {

    @Override
    public void onAddTabs(TabAdder tabs) {
        setTitle(R.string.quests);

        tabs.addTab(QuestListCursorLoader.HUB_CARAVAN, () ->
                QuestExpandableListFragment.newInstance("Village")
        );

        tabs.addTab(QuestListCursorLoader.HUB_GUILD, () ->
                QuestExpandableListFragment.newInstance("Guild")
        );

        tabs.addTab(QuestListCursorLoader.HUB_EVENT, () ->
                QuestExpandableListFragment.newInstance("Event")
        );

        tabs.addTab(QuestListCursorLoader.HUB_PERMIT, () ->
                QuestExpandableListFragment.newInstance("Permit")
        );

        super.setAsTopLevel();
    }

    @Override
    protected int getSelectedSection() {
        return MenuSection.QUESTS;
    }
}

File: app/src/main/java/com/ghstudios/android/features/quests/QuestListPagerActivity.kt
package com.ghstudios.android.features.quests

import com.ghstudios.android.AssetLoader
import com.ghstudios.android.mhgendatabase.R
import com.ghstudios.android.BasePagerActivity
import com.ghstudios.android.MenuSection
import com.ghstudios.android.data.classes.QuestHub


class QuestListPagerActivity : BasePagerActivity() {

    override fun onAddTabs(tabs: BasePagerActivity.TabAdder) {
        setTitle(R.string.title_quests)

        tabs.addTab(AssetLoader.localizeHub(QuestHub.VILLAGE)) {
            QuestExpandableListFragment.newInstance(QuestHub.VILLAGE)
        }

        tabs.addTab(AssetLoader.localizeHub(QuestHub.GUILD)) {
            QuestExpandableListFragment.newInstance(QuestHub.GUILD)
        }

        tabs.addTab(AssetLoader.localizeHub(QuestHub.EVENT)) {
            QuestExpandableListFragment.newInstance(QuestHub.EVENT)
        }

        tabs.addTab(AssetLoader.localizeHub(QuestHub.PERMIT)) {
            QuestExpandableListFragment.newInstance(QuestHub.PERMIT)
        }

        super.setAsTopLevel()
    }

    override fun getSelectedSection(): Int {
        return MenuSection.QUESTS
    }
}
